test/sdks-e2e/testdata/transformer/event-time-filter-rust.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: event-time-filter-rust
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+        transformer:
+          container:
+            # Filter messages based on event time, see https://github.com/numaproj/numaflow-go/tree/main/pkg/sourcetransformer/examples/event_time_filter
+            image: quay.io/numaio/numaflow-rust/source-transformer:stable
```

Naming - we can have multiple source transformer images in the future.
```suggestion
            image: quay.io/numaio/numaflow-rust/mapt-event-time-filter:stable
```